src/elements/content-sidebar/DetailsSidebar.js

```diff
@@ -161,9 +164,14 @@ class DetailsSidebar extends React.PureComponent<Props, State> {
         successCallback: (file: BoxItem) => void = this.fetchFileSuccessCallback,
         errorCallback: ElementsErrorCallback = this.fetchFileErrorCallback,
     ): void {
-        const { api, fileId }: Props = this.props;
+        const { api, features, fileId }: Props = this.props;
+        const archiveEnabled = isFeatureEnabled(features, 'contentSidebar.archive.enabled');
+
+        // TODO: replace this with DETAILS_SIDEBAR_FIELDS_TO_FETCH as we do not need all the sidebar fields
+        const fields = archiveEnabled ? SIDEBAR_FIELDS_TO_FETCH_ARCHIVE : SIDEBAR_FIELDS_TO_FETCH;
```

Comment:
I'm slightly concerned about the duplication in ContentSidebar. Is the former meant to preload data for the this experience? If we only add the new field here, what happens?

Reply:
I think so, file api client has some cache-ing logic ([here](https://github.com/box/box-ui-elements/blob/f50d9de11a46fca39acf56c527242fe946809991/src/api/File.js#L226-L230)), when I remove it in contentSidebar, there's some unexpected behavior with details sidebar - it goes blank for a while instead of loading (loading is shown second time I go into details tab because we already fetched all required fields):

https://github.com/user-attachments/assets/c76b68d2-75d1-4d93-a5d3-b715f89137a7

but it's side effect of it expecting only to load access stats, which has option to refresh every time we go into details sidebar.

Reply:
Should I remove it in ContentSidebar then and fix the loading?
This'll result in Archived line not being visible in loading state first time we go into details sidebar, after first load it'll start showing every time.

Reply:
Let's leave it as-is, but prioritize removing the `features` check altogether once the feature is live for everyone. That way, we can go back to using a single fields object for both calls.